code/modules/telesci/telepad.dm

```diff
@@ -116,59 +145,85 @@
 
 ///HANDHELD TELEPAD USER///
 /obj/item/weapon/rcs
-	name = "rapid-crate-sender (RCS)"
-	desc = "Use this to send crates and closets to cargo telepads."
+	name = "cargo transporter"
+	desc = "A device for teleporting crated goods."
 	icon = 'icons/obj/telescience.dmi'
 	icon_state = "rcs"
 	flags = CONDUCT
 	force = 10
 	throwforce = 10
 	throw_speed = 2
 	throw_range = 5
-	var/rcharges = 10
-	var/obj/machinery/pad = null
-	var/last_charge = 30
-	var/mode = 0
-	var/rand_x = 0
-	var/rand_y = 0
-	var/emagged = 0
-	var/teleporting = 0
-
-/obj/item/weapon/rcs/New()
-	..()
-	START_PROCESSING(SSobj, src)
+
+	var/charges = 10
+	var/maximum_charges = 10.0
+	var/obj/machinery/telepad_cargo/target_pad = null
+	var/target_turf = null
 
 /obj/item/weapon/rcs/examine(mob/user)
 	..()
-	user << "There are [rcharges] charge\s left."
+	user << "There are [charges]/[maximum_charges] charge\s left."

```

Comment:
This doesn't make sense? This is gonna be 0 to 1 charges left depending on the value.